doc/examples/color_exposure/plot_adapt_hist_eq_3d.py

```diff
@@ -0,0 +1,247 @@
+"""
+==================================
+3D adaptive histogram equalization
+==================================
+
+Adaptive histogram equalization (AHE) can be used to improve the local
+contrast of an image [1]_. Specifically, AHE can be useful for normalizing
+intensities across images. This example compares the results of applying
+global histogram equalization and AHE to a 3D image and a synthetically
+degraded version of it.
+
+.. [1] https://en.wikipedia.org/wiki/Histogram_equalization
+"""
+
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+from matplotlib import cm, colors
+from mpl_toolkits.mplot3d import Axes3D
+
+import numpy as np
+from skimage import exposure
+import imageio
+
+
+#############
+# Prepare data and apply hist. eq.
+#############
+
+im_orig = imageio.volread('https://github.com/scikit-image/'
+                          'skimage-tutorials/blob/master/'
+                          'images/cells.tif?raw=True')
```

Comment:
```suggestion
from skimage.data import image_fetcher
path = image_fetcher.fetch('data/cells.tif')
img_orig = io.imread(path) 
```

Reply:
this way we can use our new fetching mechanism with pooch, and if the data have already been downloaded once, they'll be cached for the next time they are opened.